docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -29,7 +29,7 @@ Since version 1.12, Docker also provides `overlay2` storage driver which is much
 more efficient than `overlay` in terms of inode utilization. The `overlay2`
 driver is only compatible with Linux kernel 4.0 and later.
 
-For comparison between `overlay` vs `overlay2`, please also refer to [Select a
+For comparison between `overlay` vs. `overlay2`, please also refer to [Select a
```

Comment:
I think I actually prefer 

```Markdown
For a comparison between `overlay` and `overlay2`,
please refer to ....
```

Just `For overlay vs overlay2` is not very descriptive